Sample/EventStoreDB/Simple/ECommerce.Api.Tests/ShoppingCarts/Canceling/CancelShoppingCartTests.cs

```diff
@@ -67,7 +67,7 @@ public async Task Put_Should_Confirm_ShoppingCart()
         var cartDetails = await queryResponse.GetResultFromJson<ShoppingCartDetails>();
         cartDetails.Should().NotBeNull();
         cartDetails.Id.Should().Be(fixture.ShoppingCartId);
-        cartDetails.Status.Should().Be(ShoppingCartStatus.Confirmed);
+        cartDetails.Status.Should().Be(ShoppingCartStatus.Canceled);
```

Comment:
nit: test name needs to change (Delete_Should_Cancel_ShoppingCart)
Also in tests you have some `return Task.CompletedTask` that can be removed

Reply:
Good catch 👍 Fixed in https://github.com/oskardudycz/EventSourcing.NetCore/pull/122.